scripts/upgrade.sh

```diff
@@ -116,10 +116,6 @@ function main() {
     download_util_binaries
     get_machine_id
     merge_yaml_specs
-    # Parse yaml into bash variables so we can get final kubernetes version for cases where we are upgrading 2 minor versions
```

Comment:
[This code](https://github.com/replicatedhq/kurl/commit/4d41a44e75ddc606eddc10553740dd77832b1060) is no longer necessary as it was fixed by [this commit](https://github.com/replicatedhq/kURL/commit/a6bcc0cbdc762e539134387475c4be330bc38843).